tests/conftest.py

```diff
@@ -1,8 +1,59 @@
+import pytest
+from dataclasses import dataclass
+
+
+def pytest_addoption(parser):
+    option = "--bucketfs-{name}"
+    group = parser.getgroup("bucketfs")
+    group.addoption(
+        option.format(name="port"),
+        type=int,
+        default=6666,
+        help="Port on which the bucketfs service ist listening (default: 6666)."
```

Comment:
In my experiments the default was 2580. 
Can you explain, why you have 6666?

Reply:
6666 is the port the script to start the ITDE uses. Usually easier to remember for test purposes then the official default which also wasn't the same for the docker-db all the time.